source/code/plugins/VMInsightsDataCollector.rb

```diff
@@ -141,6 +141,9 @@ def get_net_stats
 
         def get_disk_stats(dev)
             raise @baseline_exception if @baseline_exception
+            if dev.count('/') > 0
```

Comment:
Not sure why this is needed? Isn't the code being updated to always store the device names without the "/dev/" initial portion?

Reply:
In VMInsightsEngine we pass the device_name to get the disk stats. Will place the check there to pass without the dev